sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -46,7 +46,7 @@
 
 # also make regression dataset
 X_reg, y_reg = make_regression(
-    n_samples=500, n_features=10, n_informative=8, noise=10, random_state=7
+    n_samples=100, n_features=1, n_informative=8, noise=10, random_state=7
```

Comment:
`n_features=1` is a bit extreme. I would keep at least `n_features=2` to test the algorithm in a more nominal multivariate setting.

Maybe increase the `n_estimators` a bit (e.g. to 15 or 20 instead of 10) if the problem because too hard.